### PR TITLE
AC-942: Create unit test for Magento2\Less\ColonSpacingSniff check

### PR DESCRIPTION
**Pull request by svera**

### CODE REVIEW

File: Magento2/Tests/Less/ColonSpacingUnitTest.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+    /**
+     * @inheritdoc
+     */
+    public function getErrorList()
+    {
+        return [
+            8 => 1,
+            12 => 1,
+            16 => 2,
```
Comment (sivaschenko) [inline Magento2/Tests/Less/ColonSpacingUnitTest.php:18]: I believe there should be an error on line 20 as well

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the PR, @svera ! Please see my comment

File: Magento2/Tests/Less/ColonSpacingUnitTest.php (hunk 1 above)
Comment (svera) [reply Magento2/Tests/Less/ColonSpacingUnitTest.php:18]: Absolutely, good catch!

Comment (svera) [reply Magento2/Tests/Less/ColonSpacingUnitTest.php:18]: @sivaschenko the fix for the newline checking is harder than expected, as PHPCS does not respect newlines in-between instructions, so there is no way we can check this case with the current code. Should I put this as a new issue?

Comment (svera) [reply Magento2/Tests/Less/ColonSpacingUnitTest.php:18]: I've created https://github.com/magento/magento-coding-standard/issues/242

**Review by sivaschenko — APPROVED** [review]

**Review by ihor-sviziev — APPROVED** [review]

**Comment by sivaschenko** [comment]
@magento import pr to magento-commerce/magento-coding-standard

**Comment by magento-engcom-team** [comment]
@sivaschenko the pull request successfully imported.

**Pull request merged**